pandas/core/window/common.py

```diff
@@ -22,8 +22,10 @@
 
         See Also
         --------
-        Series.%(name)s : Series %(name)s.
-        DataFrame.%(name)s : DataFrame %(name)s.
+        .Series.%(name)s : Calling object with Series data.
```

Comment:
Hmm what is this for? Don't think we do this anywhere else

Reply:
What do you mean? This `_doc_template` was there before. I just improved it, IMO.

Reply:
I don’t think we need the dot prefix? Do we do this elsewhere?

Reply:
There are problems with cross-links from some modules as described in #31661. I believe this is because this module is a submodule of core, like core.window or core.groupBy and it just can't see Series/DataFrame modules as it tries to look for it in core. The dot prefix fix this. But again I don't know much about sphinx so it's only speculation.

Reply:
cc @datapythonista any ideas here?

Reply:
No, I find this quite surprising, but if that fixes all the broken links, I'd merge this, and iterate further if we better understand the problem and find a better solution in the future.

@jorisvandenbossche may be you want to have a look.

Reply:
AFAIK, the solution is to use `pandas.DataFrame....`. Not fully sure why the `.DataFrame` works, would need to look into how those names are resolved in sphinx to understand. 

@dilex42 can you try if it works correctly with adding "pandas" ?

Looking at the original example of https://github.com/pandas-dev/pandas/issues/31661 for "pipe", we actually removed "pandas." some time ago .. -> https://github.com/pandas-dev/pandas/pull/25208 So it seems that docstring validation rule SA05 is kind of wrong.

Reply:
> docstring validation rule SA05 is kind of wrong.

depending on which class it is documenting, at least. For methods on Series/DataFrame, omitting the "pandas." part works fine

Reply:
@jorisvandenbossche IIRC adding 'pandas' was working a while ago when I was testing things out. And it was also suggested as a solution in #31661.
I guess `.DataFrame` just autoprefix 'pandas' as it's top package.

Reply:
OK, but then I would prefer to be explicit and use `pandas.` instead of just the `.`

Reply:
But it explicitly adds pandas to text which can be not so 'clean' solution. It will look like `pandas.DataFrame.rolling` instead of just `DataFrame.rolling`

Reply:
That's all right. We'll also have to fix the validation in the script. If I understand correctly we should use `DataFrame.foo`, and `pandas.DataFrame.foo.bar`. If you can take care of it too, that would be great.

Reply:
If I understand correctly current validation script just look if name starts with `pandas` and mark it as error. To make it work as desired that code needs to understand from what package the reference is made and I don't have enough experience to understand how it could be done from a quick glance.

Reply:
No problem. If that check is validated, and causes the CI to fail after you make the changes, just remove its code in the validation in `ci/code_checks.sh` then. Thanks!